app/components/views/TicketsPage/PurchaseTab/PurchaseTab.module.css

```diff
@@ -87,9 +87,19 @@
   justify-self: right;
 }
 
-.isRow {
-  display: flex;
-  flex-direction: row;
+.enableVSPButton {
+  margin-bottom: 40px;
+}
+
+.enableVSPWrapper {
+  margin: 10px;
+}
+
+.enableVSPMessage {
+  background-image: var(--info-warning-icon);
```

Comment:
I dont see this icon anywhere?

![Screenshot from 2020-12-03 15-13-03](https://user-images.githubusercontent.com/6762864/101047853-133d5180-357a-11eb-8412-c2094af1eaf6.png)

Reply:
@jholdstock I think the icon makes no sense here. Instead, I fixed some minor styling issues regarding the enable message. Thanks for the suggestion!